src/common/entity/binary_sensor_icon.ts

```diff
@@ -16,9 +16,13 @@ export const binarySensorIcon = (state: HassEntity) => {
     case "garage_door":
       return activated ? "hass:garage" : "hass:garage-open";
     case "gas":
+      return activated ? "hass:shield-check" : "hass:alert";
     case "power":
+      return activated ? "hass:power-off" : "hass:power";
     case "problem":
+      return activated ? "hass:check-circle" : "hass:alert-circle";
     case "safety":
+      return activated ? "hass:check-circle" : "hass:alert-circle";
     case "smoke":
       return activated ? "hass:shield-check" : "hass:alert";
```

Comment:
```suggestion
    case "power":
      return activated ? "hass:power-off" : "hass:power";
    case "problem":
      return activated ? "hass:check-circle" : "hass:alert-circle";
    case "safety":
    case "gas":
    case "smoke":
      return activated ? "hass:shield-check" : "hass:shield-alert";
```
Maybe `fire-alert`, but fire <> smoke?